FirebaseMLModelDownloader/Sources/LocalModelInfo.swift

```diff
@@ -27,35 +27,29 @@ class LocalModelInfo {
   /// Size of the model, as returned by server.
   let size: Int
 
-  /// Local path of the model.
-  let path: String
-
-  init(name: String, modelHash: String, size: Int, path: String) {
+  init(name: String, modelHash: String, size: Int) {
     self.name = name
     self.modelHash = modelHash
     self.size = size
-    self.path = path
   }
 
   /// Convenience init to create local model info from remotely downloaded model info and a local model path.
-  convenience init(from remoteModelInfo: RemoteModelInfo, path: String) {
+  convenience init(from remoteModelInfo: RemoteModelInfo) {
     self.init(
       name: remoteModelInfo.name,
       modelHash: remoteModelInfo.modelHash,
-      size: remoteModelInfo.size,
-      path: path
+      size: remoteModelInfo.size
     )
   }
 
   /// Convenience init to create local model info from stored info in user defaults.
   convenience init?(fromDefaults defaults: UserDefaults, name: String, appName: String) {
     let defaultsPrefix = LocalModelInfo.getUserDefaultsKeyPrefix(appName: appName, modelName: name)
     guard let modelHash = defaults.value(forKey: "\(defaultsPrefix).model-hash") as? String,
-      let size = defaults.value(forKey: "\(defaultsPrefix).model-size") as? Int,
-      let path = defaults.value(forKey: "\(defaultsPrefix).model-path") as? String else {
+      let size = defaults.value(forKey: "\(defaultsPrefix).model-size") as? Int else {
```

Comment:
nit: Use https://developer.apple.com/documentation/foundation/userdefaults/1407405-integer instead of casting?

Reply:
Done!